jedi/plugins/pytest.py

```diff
@@ -181,7 +181,12 @@ def _iter_pytest_modules(module_context, skip_own_module=False):
             if Path(file_io.path) != module_context.py__file__():
                 try:
                     m = load_module_from_path(module_context.inference_state, file_io)
-                    yield m.as_context()
+                    conftest_module = m.as_context()
+                    plugins_list = m.tree_node.get_used_names().get("pytest_plugins")
+                    if plugins_list:
+                        name = conftest_module.create_name(plugins_list[0])
+                        yield from _load_pytest_plugins(module_context, name)
```

Comment:
One last question I guess: Now that we yield the plugins before the conftest module, the fixtures of the plugins will be prioritized in gotos/etc. Should it be this way or the other way around?

Reply:
Just checked it...
If a plugin and `conftest.py` have a fixture with the same name, pytest will prioritize `conftest.py`. Thus, I would turn it around.